# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../axis2c/axis2c-1.6.0-openssl-1.0.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- axis2c-src-1.6.0/src/core/transport/http/sender/ssl/ssl_utils.c.orig	2011-08-10 11:04:46.734381615 +0200
+++ axis2c-src-1.6.0/src/core/transport/http/sender/ssl/ssl_utils.c	2011-08-10 11:05:41.543932648 +0200
@@ -41,7 +41,7 @@
     axis2_char_t * key_file,
     axis2_char_t * ssl_pp)
 {
-    SSL_METHOD *meth = NULL;
+    const SSL_METHOD *meth = NULL;
     SSL_CTX *ctx = NULL;
     axis2_char_t *ca_file = server_cert;
 
